Xamarin.Forms.Platform.Android/Resources/values/styles.xml

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="UTF-8" ?>
 <resources>
-  <style name="XamarinFormsMaterialTheme" parent="Theme.MaterialComponents">
+  <style name="XamarinFormsMaterialTheme" parent="Theme.MaterialComponents.Light.DarkActionBar">
   </style>
-  <style name="XamarinFormsMaterialProgressBarHorizontal" parent="android:Widget.ProgressBar.Horizontal">
+  <style name="XamarinFormsMaterialProgressBarHorizontal" parent="Widget.AppCompat.ProgressBar.Horizontal">
```

Comment:
The reason here that I was using *android:Widget.ProgressBar.Horizontal* vs *Widget.AppCompat.ProgressBar.Horizontal*

Is that the I couldn't get the height of the progress bar to grow when it's set to *Widget.AppCompat.ProgressBar.Horizontal*

![image](https://user-images.githubusercontent.com/5375137/49984147-effbe080-ff23-11e8-8b7b-efb897458d27.png)

Reply:
I believe it has to do with this: 
https://github.com/aosp-mirror/platform_frameworks_base/blob/e80b45506501815061b079dcb10bf87443bd385d/core/res/res/values/styles_material.xml#L778-L783

We can probably override these values in our style if need be.